app/Gruntfile.js

```diff
@@ -5,9 +5,7 @@ module.exports = function(grunt) {
     pkg: '<json:package.json>',
     jshint: {
       all: {
-        src: [
-          'js/index.js', 'js/main.js'
-        ]
+        src: ['js/*.js']
```

Comment:
Nope looks good. I just couldn't remember if using wildcards like that made it check subfolders or not. Looks like it doesn't which is what we want.